data/models/user-model.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../dbConfig");
+
+async function addNewUser(user) {
+  const ids = await db("users").insert(user, "id");
+  const id = ids[0];
+  const response = await db("users")
+    .select("id", "email", "username")
+    .where({ id: id })
+    .first();
+  return response;
```

Comment:
Can you have this wrapped in a try/catch block, we need to make sure we handle errors properly

Reply:
Please do the same for all other models methods